styles.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.main {
+  background: url("images/IMG-20220819-WA0005.jpg");
+  background-repeat: no-repeat;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 100%;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.top {
+  display: flex;
+  height: auto;
+  flex-direction: column;
+  margin-top: -15px;
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.header {
+  margin: auto;
+  position: center;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-grow: 0;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.logo {
+  font-family: Poppins, Arial, Verdana, sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74px;
+  text-align: center;
+  vertical-align: center;
+  color: #6070ff;
+}
+
+a {
+  color: #6070ff;
+}
+
+.burger {
+  color: #6070ff;
+}
```

Comment:
- [ ] Please try to follow the figma design as much as possible. The button seems a little weaker than the original design. Try to make it bigger. This will enhance the appearence of your website.
Here's yours: 
![image](https://user-images.githubusercontent.com/91486791/185693858-a994921a-b2f3-48c3-a534-16fdf387b763.png)
Figma design: 
![image](https://user-images.githubusercontent.com/91486791/185693936-773afca9-e8c4-4f23-b6c0-99a9300e1b17.png)